datafusion/core/tests/sql/intersection.rs

```diff
@@ -85,3 +85,19 @@ async fn test_intersect_distinct() -> Result<()> {
     assert_batches_eq!(expected, &actual);
     Ok(())
 }
+
+#[tokio::test]
+#[should_panic(
```

Comment:
You can reference the test in https://github.com/apache/arrow-datafusion/pull/3638/files which is more elegant.

Reply:
done!